www/grpc/transaction.go

```diff
@@ -22,25 +22,20 @@ func (zs *zarbServer) GetTransaction(ctx context.Context, request *zarb.Transact
 		return nil, status.Errorf(codes.InvalidArgument, "Transaction not found")
 	}
 
-	data, err := trx.Encode()
-	if err != nil {
-		return nil, status.Errorf(codes.Internal, err.Error())
-	}
-
-	var json string
-	if request.Verbosity == 1 {
-		bz, err := trx.MarshalJSON()
-		if err != nil {
-			return nil, status.Errorf(codes.Internal, err.Error())
-		}
-		json = string(bz)
-	}
-	res := &zarb.TransactionResponse{
-		Data: data,
-		Json: json,
-	}
-
-	return res, nil
+	return &zarb.TransactionResponse{
+		Tranaction: &zarb.Transaction{
+			Id:        trx.ID().String(),
+			Version:   int32(trx.Version()),
+			Stamp:     trx.Stamp().String(),
+			Sequence:  int64(trx.Sequence()),
+			Fee:       trx.Fee(),
+			Type:      zarb.PayloadType(trx.PayloadType() - 1), //enums starting from 0
+			Payload:   trx.Payload().Signer().RawBytes(),
```

Comment:
Payload can be a [oneof](https://developers.google.com/protocol-buffers/docs/proto3#oneof) field. Therefore we can decode the payload.

Reply:
@b00f 
`oneof` field as your mentioned  clarifies payload content but then we face indexing fields of Protobuf then we might need to reserve some fields or  start using high indexes like 30 - 40 for these payloads

Reply:
@b00f 
yeah sure `oneof` field will clarify the content of payload but then we will face the indexing issue we will need to reserve bunch of fields or start indexing payload high like from 30 or so